prov/tcp/src/tcpx_init.c

```diff
@@ -56,7 +56,8 @@ int tcpx_nodelay = -1;
 
 int tcpx_staging_sbuf_size = 9000;
 int tcpx_prefetch_rbuf_size = 9000;
-
+size_t tcpx_default_tx_size;
+size_t tcpx_default_rx_size;
```

Comment:
Please initialize these to 256 (or whatever) here, so that there's always a default value present, versus delaying setting the value until checking the environment variable

Reply:
Fixed